frontend/src/common/enums/app/app-route.enum.ts

```diff
@@ -2,6 +2,7 @@ enum AppRoute {
   ROOT = '/',
   SIGN_IN = '/sign-in',
   SIGN_UP = '/sign-up',
+  EDIT_PODCAST = '/podcasts/edit/:id',
```

Comment:
```suggestion
  POCAST_EDIT_$ID = '/podcasts/edit/:id?',
```
Add `$` for dynamic params